<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="camera_position_tab">
      <attribute name="title">
       <string>Camera properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_x">
         <property name="text">
          <string>X (East)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_x">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_y">
         <property name="text">
          <string>Y (North)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_y">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_z">
         <property name="text">
          <string>Z (Elevation)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_z">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rotation_angle_tab">
      <attribute name="title">
       <string>Rotation angle</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_rot_x">
         <property name="text">
          <string>X (Angle in degrees)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_rot_x">
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_rot_y">
         <property name="text">
          <string>Y (Angle in degrees)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_rot_y">
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_rot_z">
         <property name="text">
          <string>Z (Angle in degrees)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_rot_z">
         <property name="minimum">
          <double>-360.000000000000000</double>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rotation_center_tab">
      <attribute name="title">
       <string>Rotation center</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_center_x">
         <property name="text">
          <string>X (Center)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_center_x">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_center_y">
         <property name="text">
          <string>Y (Center)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_center_y">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_center_z">
         <property name="text">
          <string>Z (Center)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBox_center_z">
         <property name="minimum">
          <double>-9999999.990000000223517</double>
         </property>
         <property name="maximum">
          <double>9999999.990000000223517</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
